### DISCUSSION TITLE
读《秋园》 | /posts/qiuy/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#592] [Announcements]
https://yuanfan.rbind.io/posts/qiuy/

**Comment by CyrusYip** [comment]
> 话说我可能是个共情能力很强的人，追剧或者看小说时情绪都会随着情节发展起起伏伏。

我也是，看到悲惨情节容易伤心难过，整个人都丧了。我觉得自己对悲伤的共情太强了，所以我都克制看悲剧或者其他难过的资讯。我和你有点不同，我对快乐的共情程度低。如果情绪 0 分是平静，50 是极乐，-50 是极悲，那我看《异人之下》难过的时候常常跌到 -50，快乐的时候也只是 20～30。

> 更不好写上周末在地摊上买了五条鱼，一个晚上过去死了四条，最后那条也没撑到中午，怀疑都是被阿木狗子吓死的。

大概是死于缺氧，没有增氧机鱼活不久。

**Reply by earfanfan** [reply]
我看《异人之下》确实是多数时候很快乐，但没有到过-50那么悲。电视剧里主要讲的罗天大醮，唯一算得上悲的就是宝宝找家人的情节，不过我知道宝宝过了那个阶段就会好了，所以也没太悲。也可能是因为毕竟被社会捶打了好多年，已经懂得安抚自己的悲伤情绪，不至于沉浸其中不可自拔了。

很久以前，读书的时候，我有个室友养鱼，就是普通的鱼缸养，一直养的很好，所以我也以为普通鱼缸就能养鱼，本来还打算再买5条的。
